webapp/src/editDogDetails/editDogDetails.tsx

```diff
@@ -535,10 +541,10 @@ const EditDogDetails = (props: any) => {
                 displayEmpty
                 renderValue={(selected: any) => (
                   <div className={classes.chips}>
-                    {(selected as string[]).map((value) => (
+                    {(selected as IBehaviour[]).map((value) => (
                       <Chip
```

Comment:
Probably revert it since group C will change their behaviours return to list of strings